src/scripts/constants.ts

```diff
@@ -436,5 +436,7 @@ export module Constants {
 	export module AriaSet {
 		export var setNameForArrowKeyNav = "setnameforarrowkeynav";
```

Comment:
I think the AriaSet module is doing more than one thing here, i.e., setNameForArrowKeyNav refers to an attribute name, while the other values are set names. I recommend keeping these in different modules.